### ISSUE TITLE
KipoiSplice/4 fails to run inside conda environment created with python 3.6

### ISSUE THREAD

**Issue by haimasree** [kipoi/models#216]
Recipe:
```
conda create --name py36 python=3.6
conda activate py36
pip install kipoi
kipoi env create KipoiSplice --verbose
......(wait a good 10/15 minutes)....
conda activate kipoi-KipoiSplice
```
Errors are - 
```
Traceback (most recent call last):
  File "test_kipoisplice.py", line 2, in <module>
    model = kipoi.get_model('KipoiSplice/4')
  File "/home/h879a/.conda/envs/kipoi-KipoiSplice/lib/python3.5/site-packages/kipoi/model.py", line 136, in get_model
    dl_source)
  File "/home/h879a/.conda/envs/kipoi-KipoiSplice/lib/python3.5/site-packages/kipoi/data.py", line 639, in get_dataloader
    CustomDataLoader = getattr(load_module(file_path), obj_name)
  File "/home/h879a/.conda/envs/kipoi-KipoiSplice/lib/python3.5/site-packages/kipoi_utils/utils.py", line 192, in load_module
    loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 697, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "dataloader.py", line 28, in <module>
  File "/home/h879a/.conda/envs/kipoi-KipoiSplice/lib/python3.5/site-packages/sklearn_pandas/__init__.py", line 3, in <module>
    from .dataframe_mapper import DataFrameMapper  # NOQA
  File "/home/h879a/.conda/envs/kipoi-KipoiSplice/lib/python3.5/site-packages/sklearn_pandas/dataframe_mapper.py", line 264
    output = [f"{name}_{o}" for o in names]
                          ^
SyntaxError: invalid syntax
```

```
2020-10-28 17:17:24.636417: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /opt/lsf/10.1/linux3.10-glibc2.17-x86_64/lib
2020-10-28 17:17:24.636521: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
ImportError: No module named 'numpy.core._multiarray_umath'
ImportError: No module named 'numpy.core._multiarray_umath'
ImportError: numpy.core._multiarray_umath failed to import
ImportError: numpy.core.umath failed to import
2020-10-28 17:17:25.047339: F tensorflow/python/lib/core/bfloat16.cc:705] Check failed: PyBfloat16_Type.tp_base != nullptr 
```

**Comment by haimasree** [comment]
Versions of  sklearn-pandas and tensorflow must be pinned to 1.8.0 and 1.13.1 respectively. These are not specified in model.yaml of KipoiSplice/4. Most likely they are coming from wither of HAL, labranchor or MaxEntScan.

**Comment by Hoeze** [comment]
Are you sure that this was created with python3.6?
/home/h879a/.conda/envs/kipoi-KipoiSplice/lib/**python3.5**/site-packages/sklearn_pandas/dataframe_mapper.py

Also, the `f"{name}_{o}"` syntax is supported from python 3.6 on.

**Comment by haimasree** [comment]
Yes I know. MaxEntScan uses python 3.5 so even if the base conda environment uses 3.6, kipoi env create KipoiSplice/4 will change the python version to 3.5. Hence the need to pin this version.

**Comment by Hoeze** [comment]
I see, i.e. the kipoi base package needs python 3.6, so we have to adjust all other models as well?

Cannot we just pin the kipoi version inside the model?

**Comment by haimasree** [comment]
Sorry I must be missing something. How will pinning kipoi resolve the fact that sklearn_pandas and tensorflow does not provide version that is meant to work with the python version that pip install is using?

**Comment by Hoeze** [comment]
It would fix that new kipoi versions need python 3.6 while old ones do not.
Basically, you could just pin every package s.t. they keep in a working state.
Updating the packages themselves would be better of course.

Or maybe I misunderstood? :D

**Comment by haimasree** [comment]
Aha! Yes that is certainly possible. How much work though, I am not sure :D
